test/e2e/specs/default-config/recordedit/add-defaults.spec.js

```diff
@@ -168,7 +177,37 @@ describe('Record Add with defaults', function() {
             expect(foreignKeyInput.getText()).toBe(values.foreign_key_value, "Foreign key input default is incorrect");
             expect(foreignKeyDisabledInput.getText()).toBe(values.foreign_key_disabled_value, "Foreign key disabled default is incorrect");
         });
-
+        //mdhelp page
```

Comment:
I know that we want to avoid having multiple test specs, but this is just too confusing for me. You added this test case to `add-defaults` specs. But this has nothing to do with "add defaults" apart from being invoked from record-edit. I think we should have a test spec for these kind of common test cases, instead of randomly adding them to a spec.

We can combine `footer`, `navbar` and also this test case into a "data-dependent" (or any other name) spec.